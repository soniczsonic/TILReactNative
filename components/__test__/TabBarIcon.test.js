import React from 'react'
import TabBarIcon from '../TabBarIcon'
import renderer from 'react-test-renderer'

// Ioniconsは、<Text />と判定されるので、このテストに意味はないが、一応記述
it('not focused home icon', () => {
  const tree = renderer
    .create(<TabBarIcon focused={false} name={'md-home'} />)
    .toJSON()

  expect(tree).toMatchSnapshot()
})
