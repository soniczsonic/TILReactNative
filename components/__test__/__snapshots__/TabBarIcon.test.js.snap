// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`not focused home icon 1`] = `<Text />`;
